src/OSPSuite.UI/Controls/UxMasterDetailGridView.cs

```diff
@@ -0,0 +1,13 @@
+using DevExpress.XtraGrid;
+
+namespace OSPSuite.UI.Controls
+{
+   public class UxMasterDetailGridView : UxGridView
+   {
+      protected override string ViewName => "UxMasterDetailGridView";
```

Comment:
The important detail is the ViewName property. That is how the grid control can create new views of the correct type. It's common to the view and a few other places in the subclass of gridcontrol.

I overrode the ViewName for UxGridView so that my subclass of grid would create UxGridView when cloning a OptimizedParameterView and I had NullReferenceExceptions from DevExpress during component resolution which were not easy to track down.